binding/flutter/CHANGELOG.md

```diff
@@ -17,4 +17,7 @@
 * Updated Voice Processor
 
 ## [1.9.9] - 2021-04-12
-* Flutter 2.0 upgrade
\ No newline at end of file
+* Flutter 2.0 upgrade
+
+## [2.0.0] - 2021-04-24
```

Comment:
We can't upgrade out Porcupine version to 2.0.0 yet, I'd revert this and I can update it.

Reply:
Okay this is reverted